slave.mk

```diff
@@ -93,7 +93,7 @@ ifneq ($(CONFIGURED_PLATFORM),generic)
 endif
 
 configure :
-	@mkdir -p $(DEBS_PATH)
+	@mkdir -p $(DEBS_PATH)/fips
```

Comment:
>fips [](http://example.com/codeflow?start=24&length=4)

Why use a subfolder?

Reply:
It is to save to the fips packages in folder, some of the packages, such as openssh-server, has two different versions, fips&none-fips one, better to place in different folders.
It will break the existing installation script (dpkg -i openssh-server*.dep), if different version of packages in the same folder.